index.html

```diff
@@ -563,6 +563,97 @@ <h3>Authorization Capability Request</h3>
     }],
     challenge: '111112b24-63d9-11ea-b99f-4f66f3e4f81a'
   }]
+}
+        </pre>
+      </section>
+    </section>
+    <section>
+      <h3>Logical Operations in Queries</h3>
+
+      <p>
+In Verifiable Presentation Requests, the structuring and retrieval of
+information rely on the use of logical operations. "AND" and "OR" operations
+play crucial roles in defining the path to desired data.
+      </p>
+
+      <section>
+        <h3>Top-Level Queries ("AND" Operation)</h3>
+        
+        <p>
+At the top-most level of our request structure, we assume an implicit "AND"
+operation between different types of queries. Each query is a unique condition
+that needs to be met to fulfill the request. This means, each query listed in
+the top-level array is an independent requirement.
```

Comment:
Here and everywhere else in this PR the "we style" language needs to be replaced. Here's a suggestion:

```suggestion
At the top-most level of the request structure, different types of queries are
expected to be processed as "AND" operations. Each query is a unique condition
that needs to be met to fulfill the request. This means, each query listed in
the top-level array is an independent requirement.
```

Also, for now, to avoid delaying this PR, I used "expected" language here instead of a normative statement (using "MUST" instead). I imagine we'll get more formal in the future in another pass to generate normative, testable statements for things like this.